src/openapi.yaml

```diff
@@ -3267,7 +3398,260 @@ components:
           type: string
           readOnly: true
           minLength: 1
+    EIOZoek:
+      type: object
+      properties:
+        uuid__in:
+          description: Array of unieke resource identifiers (UUID4)
+          type: array
+          items:
+            type: string
+            format: uuid
+        identificatie:
```

Comment:
Waar is dit attribuut voor nodig? Ik zoek toch op een lijst UUID's? Dan weet ik die identificaties niet.

Reply:
@SonnyBA Ik snap eigenlijk niet zo goed waarom generate_schema deze dingen automatisch in de openapi zet. Dit staat tevens ook zo bij de openapi van zaken-api. Snap jij dat?

Reply:
Ik vermoed dat dit een bug is in `vng-api-common`, ik zou hier iedergeval een issue voor aanmaken (en wellicht gelijk een fix?)